### ISSUE TITLE
NVDA does not announce any control including button, textbox rather group box on tab key

### ISSUE THREAD

**Issue by HK12345678** [nvaccess/nvda#9255]
When tab key pressed, NVDA does not announce anything e.g., button text.
It announces group box either.
In same PC it was working before.

### System configuration

![image](https://user-images.githubusercontent.com/47450544/52479069-62f4a100-2bcd-11e9-86be-25d7b5395ba4.png)

#### Have you tried any other versions of NVDA? Yes

**Edited information** - NVDA is being used with Windows application.

**Comment by leonardder** [comment]
Could you please run the COM Registration Fixing tool from the tools under the NVDA menu?

**Comment by HK12345678** [comment]
Thanks for the suggestion.
I tried doing this, but it did not resolve the problem. Still there is same behaviour.

**Comment by Adriani90** [comment]
@HK12345678 which application are you trying to use? Is this issue still occuring with NVDA 2020.1?

**Comment by Adriani90** [comment]
We didn't get any further updates from the initial author since year, I am closing this as abandoned. Feel free to comment on it if you are still having the issue and we can reopen.
